aiven/resource_flink_table.go

```diff
@@ -8,6 +8,30 @@ import (
 	"github.com/aiven/aiven-go-client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+const (
+	connectorTypeKafka       = "kafka"
+	connectorTypeUpsertKafka = "upsert-kafka"
+
+	formatAvro                  = "avro"
+	formatAvroConfluent         = "avro-confluent"
+	formatDebeziumAvroConfluent = "debezium-avro-confluent"
+	formatDebeziumJson          = "debezium-json"
+	formatJson                  = "json"
+
+	startupModeEarliestOffset = "earliest-offset"
+	startupModeLargestOffset  = "largest-offset"
+	startupModeGroupOffsets   = "group-offsets"
+	startupModeTimestamp      = "timestamp"
+)
+
+var (
+	aivenFlinkTableConnectorTypes    = []string{connectorTypeKafka, connectorTypeUpsertKafka}
+	aivenFlinkTableKafkaStartupModes = []string{startupModeEarliestOffset, startupModeLargestOffset, startupModeGroupOffsets, startupModeTimestamp}
+	aivenFlinkTableKafkaValueFormats = []string{formatAvro, formatAvroConfluent, formatDebeziumAvroConfluent, formatDebeziumJson, formatJson}
+	aivenFlinkTableKafkaKeyFormats   = aivenFlinkTableKafkaValueFormats
```

Comment:
Why do we need two variables with the same values?

Reply:
they have different semantic and only coincidentally the same values i think, i'm for separating the names

Reply:
:heavy_check_mark: